Source/Core/DolphinQt2/GameList/TableDelegate.cpp

```diff
@@ -52,24 +58,44 @@ void TableDelegate::paint(QPainter* painter, const QStyleOptionViewItem& option,
 
 QSize TableDelegate::sizeHint(const QStyleOptionViewItem& option, const QModelIndex& index) const
 {
+  const auto scale = DPI::GetScale();
   switch (index.column())
   {
   case GameListModel::COL_PLATFORM:
-    return Resources::GetPlatform(0).size();
+    return Resources::GetPlatform(0).size() / Resources::GetIconScale() * scale;
```

Comment:
Why the ` / Resources::GetIconScale() * scale`? There's a chain of `GetIconScale()`, `GetScaledIconSize()`, and `GetScale()` that makes zero sense to me here.

Just use `pixmap.size() / pixmap.devicePixelRatio()`, which, with `AA_EnableHighDpiScaling` set, will always work.

Also, I know this is building off of the way it was before, but this logic is starting to get non-simple enough that I think it should be extracted into the `Resources` namespace, something like `Resources::GetPlatformPixmapLayoutSize`.